src/services/item/plugins/importExport/index.ts

```diff
@@ -101,7 +102,7 @@ const plugin: FastifyPluginAsync = async (fastify) => {
   );
 
   // download item as zip
-  fastify.route<{ Params: { itemId: string } }>({
+  fastify.route<{ Params: { itemId: string }; Querystring: { actionType?: string } }>({
```

Comment:
Might be interesting to add a comment about what values can this be? Will it refer to where in the ui we clicked? Can this info be added in the extra and have a global action type?